README.md

```diff
@@ -27,3 +28,6 @@ Run `make help` to check available commands.
 ## License
 
 [Apache 2.0 License](../LICENSE).
+
```

Comment:
We need to add this to the template’s readme otherwise it will be lost. I can do it tomorrow.

Reply:
Nevermind 😅 this is the main README, not the chart readme.